geonode/layers/utils.py

```diff
@@ -375,7 +377,8 @@ def get_bbox(filename):
         bbox_y0 = min(ext[0][1], ext[2][1])
         bbox_x1 = max(ext[0][0], ext[2][0])
         bbox_y1 = max(ext[0][1], ext[2][1])
-        srid = srs.GetAuthorityCode(None) if srs else 'EPSG:4326'
+        # eliminate default EPSG srid as it will be added when this function returned
+        srid = srs.GetAuthorityCode(None) if srs else '4326'
 
     return [bbox_x0, bbox_x1, bbox_y0, bbox_y1, "EPSG:%s" % str(srid)]
```

Comment:
even an expert geonode developer can miss this :joy:

Reply:
This should have been tested by unittest. If not, then we should add.

Reply:
Also we need to submit PR to upstream GeoNode for issues like this.

Reply:
@boney-bun cc @lucernae we should be doing all GeoNode PRs directly upstream.

Also why is this merged when we decided to not assume 4326 if no projection defined?